### ISSUE TITLE
Unknown command 'saveenv' after flashing 

### ISSUE THREAD

**Issue by ThorodanBrom** [parallella/parallella-flash#2]
There may be a problem with the flashed version of U-Boot, it may have been compiled without saveenv @olajep . Was that on purpose?

**Comment by ThorodanBrom** [comment]
Is this because all environment variables are just loaded from uEnv.txt?

**Comment by MatthiasPopp** [comment]
I just updated my board from the 2014.1 (preinstalled) to parallella-next (latest version)
Both uboot instances came with the saveenv command.  
Best regards
Matthias

**Comment by MatthiasPopp** [comment]
I had to redo it because some environment parameter are missing.
You are right the make target from the uboot is lacking the features. I do not see how this is compiled in afterwards.

**Comment by ThorodanBrom** [comment]
Yeah, without `saveenv`, the only option as far as I remember is to redo everything
